conductor-client/src/test/kotlin/com/openlattice/data/LocalBlobDataServiceTest.kt

```diff
@@ -70,7 +71,7 @@ class LocalBlobDataServiceTest {
         }
         key1 = key1.plus(data.hashCode())
 
-        byteBlobDataManager.putObject(key1, data, "png")
+        byteBlobDataManager.putObject(key1, BinaryDataWithMetadata("png", data))
```

Comment:
`"png"` is not a valid content-type, though not sure if it matters in the test.

https://www.iana.org/assignments/media-types/media-types.xhtml#image

```suggestion
        byteBlobDataManager.putObject(key1, BinaryDataWithMetadata("image/png", data))
```

Reply:
i don't think it matters, this test has existed for forever

Reply:
In that case, it doesn't hurt to hit that "Commit suggestion" button.

<img width="362" alt="Screen Shot 2021-04-07 at 9 42 13 AM" src="https://user-images.githubusercontent.com/440299/113903184-8a336280-9785-11eb-92e7-87c29c58bf60.png">